scripts/gatsby-source-iiif/setupEnv.js

```diff
@@ -26,6 +28,8 @@ const retrieveStageParameters = async () => {
       AUTH_CLIENT_ID: '0oa1f3ut0aKpdwap5357',
       AUTH_CLIENT_ISSUER: 'https://okta.nd.edu/oauth2/ausxosq06SDdaFNMB356',
       S3_DEST_BUCKET: '',
```

Comment:
if we want to do this for completeness this line should probably be something like process.env.S3_DEST_BUCKET since it is in the env and is used in the config file.  as process.env.